new-docs/layouts/shortcodes/literalInclude.html

```diff
@@ -0,0 +1,124 @@
+{{ $dynPath := .Params.dynPath }}
```

Comment:
Could you add some docs in here about how to use this? Similar to what was in your standup update this morning.

Reply:
8b9fd7b — can add more after vacation, but hopefully that's useful enough for now 😄